src/content/content.js

```diff
@@ -14,18 +14,18 @@ let uiController = null
 
 let sendContentReadyToBackground = function sendContentReadToBackground () {
   messagingService.sendMessageToBg(new ContentReadyMessage(uiController.state))
-    .catch((error) => console.error('Unable to send content ready message to background', error))
+    .catch((error) => console.error('Alpheios error: Unable to send content ready message to background', error))
 }
 
 let sendStateToBackground = function sendStateToBackground () {
   messagingService.sendMessageToBg(new StateMessage(uiController.state))
-    .catch((error) => console.error('Unable to send state to background', error))
+    .catch((error) => console.error('Alpheios error: Unable to send state to background', error))
 }
 
 let sendResponseToBackground = function sendResponseToBackground (request, state = uiController.state) {
   messagingService.sendResponseToBg(new StateResponse(request, state)).catch(
     (error) => {
-      console.error('Unable to send a response to a state request', error)
+      console.error('Alpheios error: Unable to send a response to a state request', error)
```

Comment:
@balmas , sometimes you use prefix `Alpehios warning` or `Alpheios error`, sometimes not.
May be it should be the same way?

Reply:
I do think that adding the info about the domain from where the message has been originated (`Alpheios`) is beneficial as it enables us to filter out Alpheios message only. On the other side, it is good to keep the message short: it will make it easier to understand. I'm not sure if we need to add a word `error` here as the color of the message in the console will indicate what type of error it is. Removing it will shorten the message. However, if error log is to be copied and stored somewhere, it's good to have an message type in the message text so that it will be stored and analyzed later. I'm not sure which variant is better. What do you think?

I also think that we can use the logger library here, and make it add an Alpheios prefix that we'll decide upon into the message text automatically. Then we won't have to add it manually to every message. What is your opinion on that?

I also do agree with @irina060981 comment about the format consistency.